test/fixtures/resources.json

```diff
@@ -2,7 +2,7 @@
   "_copyright": "Copyright (C) 2012 - 2013, Digium, Inc.",
   "_author": "David M. Lee, II <dlee@digium.com>",
   "_svn_revision": "$Revision$",
-  "apiVersion": "1.7.0",
+  "apiVersion": "3.0.0",
```

Comment:
is this generated or did you mean to make it 3.0.0 instead of 2.0.0?

Reply:
It is generated yes. We're simply loaded the files from Asterisk and storing them as fixtures.